parachains/integration-tests/collectives/1_alliance/3_kick_member.yml

```diff
@@ -129,13 +120,26 @@ tests:
               pallet: xcmPallet
               call: send
               args: [
-                { v1: { parents: 0, interior: { x1: { parachain: *cp_id }}}}, # destination
+                { v3: { parents: 0, interior: { x1: { parachain: *cp_id }}}}, # destination
                 {
-                  v2: [ #message
+                  v3: [ #message
+                    {
+                      UnpaidExecution: {
+                        weightLimit: { 
+                          limited: {
+                            refTime: 4000000000, # 3_000_000_000
```

Comment:
The comment here seems inconsistent with the value

Reply:
think we can ditch these comments - they don't add that much.